web/src/screens/private/Dashboard.js

```diff
@@ -3,38 +3,120 @@ import PropTypes from 'prop-types';
 import { compose } from 'recompose';
 import { graphql } from 'react-apollo';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import moment from 'moment';
 
 import { withStyles } from 'material-ui/styles';
 import { CircularProgress } from 'material-ui/Progress';
 import Paper from 'material-ui/Paper';
 import Typography from 'material-ui/Typography';
+import Toolbar from 'material-ui/Toolbar';
+import Table, { TableBody, TableCell, TableRow, TableHead } from 'material-ui/Table';
 
 import CURRENT_USER_QUERY from '../../graphql/current-user.query';
 
 import styles from './Dashboard.styles';
 
-const Dashboard = ({ classes, loading }) => {
+
+const displayRequests = (classes, user) => {
+  if (!user.groups.length) {
+    return (
+      <Paper className={classes.paper}>
+        <Typography component="p" align="center">
+          You are currently not assigned to any groups. <Link to="/groups">Click here to join groups</Link>
+        </Typography>
+      </Paper>
+    );
+  }
+
+  return (
+    <Paper className={classes.paper}>
+      <Toolbar className={classes.tableToolbar}>
+        <Typography variant="title">Requests</Typography>
+      </Toolbar>
+      <Table className={classes.table}>
+        <TableHead>
+          <TableRow>
+            <TableCell>Name</TableCell>
+            <TableCell>Group</TableCell>
+            <TableCell>Type</TableCell>
+            <TableCell>Start Date</TableCell>
+            <TableCell>End Date</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {user.schedules.map(schedule => (
+            <TableRow key={schedule.id}>
+              <TableCell>
+                <Link to={`/schedules/${schedule.id}`}>{schedule.name}</Link>
+              </TableCell>
+              <TableCell>{schedule.group.name}</TableCell>
+              {/* TODO: Make sure schedule object has this in the future */}
+              {/* <TableCell>{schedule.type}</TableCell> */}
+              <TableCell>
+                {schedule.startTime === 0 ? '-' : moment.unix(schedule.startTime).format('LLL')}
+              </TableCell>
+              <TableCell>
+                {schedule.endTime === 2147483647
```

Comment:
can we toss this in a `constants.js` or something? make one for `0` too if you're feeling game. Later on I hope we'll create a common lib for this kinda stuff (shared amongst web, client, server etc).